### PR TITLE
only check external hrefs on master

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,6 +34,13 @@ TEST_DIRS     ?= $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
                    -exec dirname {} \\; | sort | uniq")
 VERSION       ?= $(shell git describe --always --abbrev=7 --dirty)
 BUILD_LDFLAGS  = $(shell build/version.sh $(ROOT) $(SC_PKG))
+GIT_BRANCH    ?= $(shell git branch 2> /dev/null | grep "^\*" | sed "s/^\* *\(.*\)/[\1]/")
```
Comment (MHBauer) [inline Makefile]: what is the logic in here doing? look for a star and then do a bunch of sed stuff?

Comment (duglin) [reply Makefile]: for me "git branch" will show the current branch with a "*" before it, so this just removes that. If there's a better (easier) way to get the current branch I'm ok with switching.

Comment (MHBauer) [reply Makefile]: I think the classic is `git rev-parse --abbrev-ref HEAD` if you want the name.

Comment (duglin) [reply Makefile]: ooo I like that better - switching....

**Pull request merged**